tests/test_keras_integration.py

```diff
@@ -180,6 +180,35 @@ def model_fit(self, model):
         model.fit(x, y)
         return model
 
+    def model_fit_callback(self, model, callback):
+        x = tf.constant([[0.44, 0.90], [0.65, 0.39]])
+        y = tf.constant([[1, 1], [0, 0]])
+        model.fit(x, y, callbacks=[callback])
+        return model
+
+    def test_callback(self):
+        REPO_NAME = repo_name("PUSH_TO_HUB")
+        model = self.model_init()
+
+        push_to_hub_callback = PushToHubCallback(
+            save_strategy="epoch",
```

Comment:
can we make sure we're testing all strategies?

Reply:
I will write the tests at the end because I don't know how to test callbacks, I have to check my options and decide. The behavior here we want to test are if they're really saved and pushed in the periods given by user periodically, not only at the end of training so I somehow need to log this info, either through commit history or inside the training.